app.rb

```diff
@@ -1,9 +1,24 @@
 require 'sinatra/base'
+require './lib/chitter_homepage'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/chitter' do
+    @peeps = ChitterPage.view
+    erb :peeps
+  end
+
+  get '/chitter/new' do
+    erb :new_message
+  end
+
+  post '/chitter/create' do
```

Comment:
When you use 'post' you can use the same route name, this helps to have a restful api.